PROTOCOL.md

```diff
@@ -117,6 +117,58 @@ Response:
 }
 ```
 
+### getObjectProperty
+
+```json
+{
+    "callback":false,
+    "id":134,
+    "method":"getObjectProperty",
+    "params":[
+        14,
+        "itemIndex"
+    ]
+}
+```
+
+Response:
+
+```json
+{
+    "id": 134,
+    "result": 1
+}
+```
+
+### callObjectMethod
+
+```json
+{
+    "callback":{},
+    "id":19,
+    "method":"callObjectMethod",
+    "params":[
+        {
+            "lazarusObjectId":14,
+            "method":"items.addObject",
+            "params":[
+                "Male",
+                0
+            ]
+        }
+    ]
```

Comment:
@reisraff: Can you change for this (see below)? It's more compact ;-)

"method":"callObjectMethod",
"params":[
	14,
	"items.addObject",
	[
		"Male",
		0
	]
]